packit/sync.py

```diff
@@ -0,0 +1,47 @@
+import glob
+import os
+import shutil
+import logging
+
+from packit.exceptions import PackitException
+from packit.config import PackageConfig, SyncFilesItem, SyncFilesConfig
+
+logger = logging.getLogger(__name__)
+
+
+def get_wildcard_resolved_sync_files(pc: PackageConfig) -> None:
+    logger.debug("Packit synced files %s", pc.synced_files.files_to_sync)
+    files_to_sync = []
+    for sync in pc.synced_files.files_to_sync:
+        if isinstance(sync, str):
```

Comment:
it means that globbing is allowed just for `src, dest` values, in case there will be "plain" `- *.spec` it will try to copy `"*.spec"` file. It is okay, but it is documented that globbing is intended just fot these items?

Reply:
Will try to fix it.

Reply:
There is already check for this https://github.com/packit-service/packit/pull/205/files#diff-15499454dfa98296a4dcf92153faca12R577 . And it works.